doc/7/controllers/collection/delete/index.md

```diff
@@ -0,0 +1,32 @@
+---
+code: true
+type: page
+title: delete
+description: Delete a collection
+---
+
+# delete
+
+Deletes a collection.
+
+<br/>
+
+```js
+delete(index, collection, [options]);
```

Comment:
What kind of `options` can you pass? Also you should add this to the arguments table